CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Ongoing
+
+- Fix TODO #294
```

Comment:
Probably adjust to 'repair coverage/fix testing'?

Reply:
Sure, no problem.